### DISCUSSION TITLE
btw heres wifi password if u rowland schools

### DISCUSSION THREAD

**Discussion by testacc992** [3kh0/ext-remover#498] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]

**Comment by Aka-but-nice** [comment]
Imagine clicking the link

**Reply by testacc992** [reply]
I wonder who clicked on it 💀 
![image](https://user-images.githubusercontent.com/94496154/218009128-6e7de8cf-613d-4970-88ce-180aef682d5f.png)

**Comment by Aka-but-nice** [comment]
You can literally see where it goes to, don't fall for this, even if you're in the same school as this supposedly person.

**Comment by amelia-bartlett** [comment]
> btw heres wifi password if u rowland schools -roka
> 
> ![Screenshot 2023-02-03 11 49 52 AM](https://user-images.githubusercontent.com/94496154/217163085-11426ef0-beda-422d-88e7-d7d876bae256.png)
> 
> websites link: [https://bit.ly/rokasfunnyprank](url)

I probably wouldn't post your school's wifi password here, this is a very good way to bring the administrators down on you.

You would be better off privately sharing it rather than leaking it, also it may get changed if the admin finds out.

**Comment by obkduishinijf** [comment]
> btw heres wifi password if u rowland schools -roka
> 
> ![Screenshot 2023-02-03 11 49 52 AM](https://user-images.githubusercontent.com/94496154/217163085-11426ef0-beda-422d-88e7-d7d876bae256.png)
> 
> websites link: [https://bit.ly/rokasfunnyprank](url)

bro think we dumb enough to click the link, it just takes you to an error page on gyazo though

**Reply by amelia-bartlett** [reply]
hang on, imma see what cURL says about it
yeah its a redirect to "gyazo.nl", which has nothing (cURL wasn't set to output HTTP status codes)
![image](https://user-images.githubusercontent.com/44680463/217928666-2e73574e-b084-40ff-820c-d14ed994b687.png)

when trying to load in browser, an extension kills the request as an ip grabber
![image](https://user-images.githubusercontent.com/44680463/217929001-b450bc01-e11d-46ba-a8b5-ac56c76d4743.png)

potentially an admin trying to figure out who is using sh1mmer with a bait-and-switch?

**Reply by Aka-but-nice** [reply]
> hang on, imma see what cURL says about it yeah its a redirect to "gyazo.nl", which has nothing (cURL wasn't set to output HTTP status codes) ![image](https://user-images.githubusercontent.com/44680463/217928666-2e73574e-b084-40ff-820c-d14ed994b687.png)
> 
> when trying to load in browser, an extension kills the request as an ip grabber ![image](https://user-images.githubusercontent.com/44680463/217929001-b450bc01-e11d-46ba-a8b5-ac56c76d4743.png)
> 
> potentially an admin trying to figure out who is using sh1mmer with a bait-and-switch?

wth is that

**Reply by amelia-bartlett** [reply]
> > hang on, imma see what cURL says about it yeah its a redirect to "gyazo.nl", which has nothing (cURL wasn't set to output HTTP status codes) ![image](https://user-images.githubusercontent.com/44680463/217928666-2e73574e-b084-40ff-820c-d14ed994b687.png)
> > when trying to load in browser, an extension kills the request as an ip grabber ![image](https://user-images.githubusercontent.com/44680463/217929001-b450bc01-e11d-46ba-a8b5-ac56c76d4743.png)
> > potentially an admin trying to figure out who is using sh1mmer with a bait-and-switch?
> 
> wth is that

cURL is a command line program to download over HTTP (often used in scripts)

the screenshot shows an error page from a browser extension called "Universal Bypass" which aims to locally handle link shorteners, adfly, and similar.  In this case, this error is triggered by such a shortener that cannot be bypassed, and is also an ip logger.

**Reply by amelia-bartlett** [reply]
also, forgot to add that by "the screenshot" i mean the second screenshot (the first one shows the output of cURL)

**Comment by Aka-but-nice** [comment]
Does this language, also use HTML, it seems as this is true, I see a lot of html
